mopidy/backend.py

```diff
@@ -228,9 +228,22 @@ def is_live(self, uri):
         """
         return False
 
+    def should_download(self):
```

Comment:
Part of me think we should pass the uri in here too. But we can solve that another time,